lib/tasks/db.js

```diff
@@ -437,6 +465,59 @@ function getMigrator (type) {
   }
 }
 
+function runQueryOnDefaultDatabase(query, config) {
+  var dialect = config.dialect;
+
+  return new Bluebird(function () {
+    if (dialect.match('mysql')) {
+      var mysql = require('mysql');
+      var connection = mysql.createConnection({
+        host: config.host,
+        user: config.username,
+        password: config.password,
+        port: config.port
+      });
+
+      connection.query(query, function (error) {
+        if (error) {
+          console.error(error);
+          process.exit(1);
+        }
+        else {
+          process.exit(0);
+        }
+      });
+    }
+    else if (dialect.match('postgres')) {
+      var Postgres = require('pg').Client;
+      var client = new Postgres({
+        user: config.username,
+        host: config.host,
+        database: 'postgres',
```

Comment:
`config.database`?

Reply:
@dagumak nope, I am passing `postgres` because the database doesn't exist yet, so I am using the default one.